import 'nanopb.proto';

package usg.Events;

message RequestDebugCameraState
{
	required bool bEnable = 1;
}


message RequestDebugCameraState_Internal
{
	required bool bEnable = 1;
}